setup.py

```diff
@@ -123,7 +123,7 @@ def _read(file):
 class pil_build_ext(build_ext):
     class feature:
         features = ['zlib', 'jpeg', 'tiff', 'freetype', 'lcms', 'webp',
-                    'webpmux', 'jpeg2000', 'imagequant']
+                    'webpmux', 'jpeg2000', 'imagequant', 'dxtc']
```

Comment:
There is no need to create a separate feature, because `_dxtc.c` doesn't depend on external modules which can be not available.